apps/web/src/components/commands/nft/Config.tsx

```diff
@@ -19,17 +19,18 @@ export const NftConfig = (props: CommandExtensionProps<NftWidgetProps>) => {
       chain: formData.get("chain")
         ? Number(formData.get("chain")?.toString())
         : undefined,
+      tokenId: Number(formData.get("tokenId")?.toString()) ?? undefined,
     });
 
     setOpen(false);
 
     props.editor.view.dom.focus();
   }
 
-  const { property, address, chain } = props.node.attrs;
+  const { property, address, chain, tokenId } = props.node.attrs;
 
   const isConfigured =
-    property !== undefined && address !== undefined && chain !== undefined;
+    property !== undefined && address !== undefined && chain !== undefined || tokenId !== undefined
```

Comment:
Hmm, seems a little hacky. Would be nice in the future to separate the NFT commands configuration. A strategy pattern could solve it.